src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robotDublicate = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (!(value in robotDublicate)) {
+      robotDublicate[value] = key;
+    } else {
+      return null;
+    }
```

Comment:
invert this condition, so you wouldn't need to use `else` here
```suggestion
    if ((value in robotDublicate)) {
      return null;
    }
    
     robotDublicate[value] = key;
```